client/src/components/reports/statistics/DetailedItemStatistics.tsx

```diff
@@ -0,0 +1,131 @@
+import React, {useMemo} from 'react';
```

Comment:
Might just be me, but when I first login and then select Detailed Item Statistics, nothing shows up unless I've selected Item Demand Report first.

Reply:
It's not just you, I need to figure out what's causing that